tests/test_student.py

```diff
@@ -0,0 +1,10 @@
+import pytest
+from flask import g, session
+from portal.db import get_db
+
+def test_student_home(client, auth):
+    response =  client.get('/student/home')
+    assert 'http://localhost/' == response.headers['Location']
+    auth.student_login()
+    response = client.get('/student/home')
+    assert b'<li>CSET</li>' in response.data
```

Comment:
This test asserts data from a placeholder will exist, so, while it's currently passing, it will fail when we replace the placeholder data in portal/students.py with real sessions